src/styles/app/25-node-note-content.css

```
@@ -28,46 +28,84 @@
 .wikiLinkIcon {
 	--wiki-link-file-icon-mask: url("data:image/svg+xml,%3Csvg%20xmlns%3D'http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg'%20viewBox%3D'0%200%2024%2024'%20fill%3D'none'%3E%3Cpath%20d%3D'M12.8809%207.01656L17.6538%208.28825M11.8578%2010.8134L14.2442%2011.4492M11.9765%2017.9664L12.9311%2018.2208C15.631%2018.9401%2016.981%2019.2998%2018.0445%2018.6893C19.108%2018.0787%2019.4698%2016.7363%2020.1932%2014.0516L21.2163%2010.2548C21.9398%207.57005%2022.3015%206.22768%2021.6875%205.17016C21.0735%204.11264%2019.7235%203.75295%2017.0235%203.03358L16.0689%202.77924C13.369%202.05986%2012.019%201.70018%2010.9555%202.31074C9.89196%202.9213%209.53023%204.26367%208.80678%206.94841L7.78366%2010.7452C7.0602%2013.4299%206.69848%2014.7723%207.3125%2015.8298C7.92652%2016.8874%209.27651%2017.2471%2011.9765%2017.9664Z'%20stroke%3D'black'%20stroke-width%3D'1.5'%20stroke-linecap%3D'round'/%3E%3Cpath%20d%3D'M12%2020.9462L11.0477%2021.2055C8.35403%2021.939%207.00722%2022.3057%205.94619%2021.6832C4.88517%2021.0607%204.52429%2019.692%203.80253%2016.9546L2.78182%2013.0833C2.06006%2010.3459%201.69918%208.97718%202.31177%207.89892C2.84167%206.96619%204%207.00015%205.5%207.00003'%20stroke%3D'black'%20stroke-width%3D'1.5'%20stroke-linecap%3D'round'/%3E%3C%2Fsvg%3E")
 		center / contain no-repeat;
 
 	flex: 0 0 auto;
-	width: var(--icon-md);
-	height: var(--icon-md);
-	background: var(--text-accent);
-	color: var(--text-accent);
+	width: var(--icon-sm);
+	height: var(--icon-sm);
+	background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)